src/core.js

```diff
@@ -149,6 +168,8 @@ c3_chart_internal_fn.initWithData = function (data) {
     var $$ = this, d3 = $$.d3, config = $$.config;
     var defs, main, binding = true;
 
+    $$._axis = new _Axis($$);
```

Comment:
I think it's better code manner to add some prefix like underscore for emphasizing a intention  "this is internal and don't use from outside of the module!" and '$$' is ambiguous for that purpose...but I'll remove them anyway.